protostar/commands/cairo1_commands/init_command.py

```diff
@@ -37,12 +37,22 @@ def arguments(self):
                 type="str",
                 is_positional=True,
             ),
+            ProtostarArgument(
+                name="minimal",
+                type="bool",
+                description="Create a minimal project structure.",
+            ),
         ]
 
     async def run(self, args: Namespace):
-        self.init(project_name=args.name)
+        minimal = False
+        if vars(args).get("minimal"):
+            minimal = bool(args.minimal)
```

Comment:
`ProtostarArgument` should accept an `default=` when constructing. There's no need to solve it like this.

Reply:
[feedback](https://github.com/software-mansion/protostar/pull/1958/commits/095daf8b6e677d4336582679a059432726939c90)